src/index.js

```diff
@@ -0,0 +1,55 @@
+// import _ from 'lodash';
+import './style.css';
+
+// function component() {
+//   const element = document.createElement('div');
+
+//   // Lodash, now imported by this script
+//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+//   element.classList.add('hello');
+//   return element;
+// }
+
+// document.body.appendChild(component());
+
+const todoList = document.getElementById('todoList');
+// window.addEventListener('load', () => {
+//     todoList.style.display= "block";
+//     }
+// );
```

Comment:
- [ ] **_Required:_** kindly remove any unnecessary comments to make your code looks clean 🙏